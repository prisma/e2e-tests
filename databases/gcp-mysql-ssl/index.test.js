const { PrismaClient, Prisma } = require('@prisma/client')
const fs = require('fs')
const prisma = new PrismaClient()

const pjson = require('./package.json')

describe('tests for GCP MySQL SSL database', () => {
  afterAll(() => {
    prisma.$disconnect()
  })

  it('should test the Prisma version', async () => {
    expect(Prisma.prismaVersion.client).toMatchSnapshot(
      pjson['dependencies']['@prisma/client'],
    )
  })

  it('should query the database', async () => {
    const data = await prisma.user.findMany({
      where: { email: 'alice@prisma.io' },
      select: { email: true, name: true },
    })
    expect(data).toMatchSnapshot()
  })
})
