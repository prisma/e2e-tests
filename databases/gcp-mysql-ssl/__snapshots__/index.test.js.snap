// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for GCP MySQL SSL database should query the database 1`] = `Array []`;

exports[`tests for GCP MySQL SSL database should test the Prisma version: 2.14.0-dev.5 1`] = `"2.13.0-dev.101"`;
